feature/Initial-UI-setup

Saafine opened the pull request.

allgreed commented on .gitignore:
```diff
@@ -0,0 +1 @@
+.idea
```
Please use [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) for private tool artifacts

Saafine replied:
Done

allgreed commented on ui/.gitignore:
```diff
@@ -0,0 +1,23 @@
+pnpm-debug.log*
+
+# Editor directories and files
+.idea
```
Please use [global gitignore](https://gist.github.com/subfuzion/db7f57fff2fb6998a16c) for private tool artifacts

Saafine replied:
Done, good idea

allgreed commented on ui/.gitignore:
```diff
@@ -0,0 +1,23 @@
+.env.*.local
+
+# Log files
+npm-debug.log*
```
Which one of those are you actually using? Could you please remove the others?

Saafine replied:
Removed

allgreed commented on ui/.gitignore:
```diff
@@ -0,0 +1,23 @@
+
+
+# local env files
+.env.local
```
Are you actually using those? I don't see a `.env`

Saafine replied:
Done

allgreed commented on ui/.gitignore:
```diff
@@ -0,0 +1,23 @@
+.DS_Store
```
I hereby accept the fact that you're using MacBook, would you kindly move this to global gitignore? ^^

Saafine replied:
Done

allgreed commented on ui/README.md:
```diff
@@ -0,0 +1,24 @@
+# clustogram
```
I highly recommend sticking to some form of [my README template](https://github.com/allgreed/templates/blob/master/README.md_template), however I'm not having super strong opinion about this. Just please make sure that relevant `Makefile` targets work. It's easiest to achieve the make <-> README parity by simply having README redirect to make, hence my suggestion. But as long as Make works as expected you do you ;)

Saafine replied:
I removed README from ui folder. I think it's not necessary if README in root directory exists already.

allgreed reviewed (changes requested):
Nice work, and thank you for the first contribution to this project. I may seem nitpicky, but it's mostly to set standards and provide a case study for your teammates. I believe the comments can be addressed in less than 15 minutes and then I'll gladly merge it.

Also: could you please please provide working `default.nix` in the next few iterations? It's a soft requirement for getting all components integrated.

Saafine commented:
> Nice work, and thank you for the first contribution to this project. I may seem nitpicky, but it's mostly to set standards and provide a case study for your teammates. I believe the comments can be addressed in less than 15 minutes and then I'll gladly merge it.
> 
> Also: could you please please provide working `default.nix` in the next few iterations? It's a soft requirement for getting all components integrated.

Thank you for the comments,
1) default.nix should be now fixed.
2) I updated my global .gitignore
3) I have a question to Makefile:

Do you think we need these commands too? I am not exactly sure what to put there:

`env-up: ## set up dev environment
	@echo "Not implemented"; false

env-down: ## tear down dev environment
	@echo "Not implemented"; false

env-recreate: env-down env-up ## deconstruct current env and create another one

container: build ## create container
	#docker build -t lmap .
	@echo "Not implemented"; false
`
